.nvmrc

```diff
@@ -1 +1 @@
-v8.10.0
+v10.0.0
```

Comment:
At one point, this was set to `lts`. Is there a reason not to do that? Current LTS is v10. I know you can use that shorthand in nvmrc. Not sure about the other configs.

Reply:
LTS sounds nice, but it's not very deterministic. What if this blows up our build in November (when node 12 becomes LTS) and we need to react vs. planning the work?